adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +106,56 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    [GADMediationAdapterPangle setCOPPA:adConfiguration.childDirectedTreatment.integerValue];
```

Comment:
Update to: 
```
    [GADMediationAdapterPangle setCoppa:adConfiguration.childDirectedTreatment];
```

Reply:
I update this invoke way ：
if (adConfiguration.childDirectedTreatment) { [GADMediationAdapterPangle setCOPPA:adConfiguration.childDirectedTreatment.integerValue]; }

Reply:
I update this invoke way ：
`if (adConfiguration.childDirectedTreatment) { [GADMediationAdapterPangle setCOPPA:adConfiguration.childDirectedTreatment.integerValue]; }`